### ISSUE TITLE
TRON ETH HTTP API returning NIL response for eth_TransactionByReceipt

### ISSUE THREAD

**Issue by panbhatt** [tronprotocol/java-tron#4969]
### System information

java-tron version: `java -jar FullNode.jar -v`
OS & Version: UBUNTU

we are trying to hit the HTTP ETH API to get the data of various blocks and transactions. Unfortunately, for some of the initial blocks we are getting NO data for eth_getTransactionReceipt for some of the transactions.
### 1. What did you do?
Hit the API request via POSTMAN. Tx is : 0x5281eb99b3cab6b3cb4777e6b070df936f2cb068163fb8f3959a1cd88733a4c2 
![image](https://user-images.githubusercontent.com/1473470/217078275-f41a03f0-6add-45ea-be65-da42632abb04.png)

### 2. What did you expect to see?
All the data about this transaction. 

### 3. What did you see instead?
Result is NULL, which is not expected. 
Please can you help to determine what is the root cause here. Are we expected to get any data back or not. For All the other transactions we can see this API responds perfectly. 

This same is happening for another 6-7 transactions in the initial blocks from 1-1000.

**Comment by sunflower-sun** [comment]
eth_getTransactionReceipt is used to query the transactionInfo: fee, result .... but at the beginning TRON does not support transactionInfo, so for the transactions in some blocks earlier in the blockchain，they do not have transactioninfo. but you can get the detailed information through eth_getTransactionByHash

**Comment by nateless** [comment]
@sunflower-sun do you know from which block it started to provide receipts?

**Comment by panbhatt** [comment]
I will soon provide this information, we are trying to index the TRON Server.

**Comment by sunflower-sun** [comment]
> @sunflower-sun do you know from which block it started to provide receipts?

That depends the node, After the transaction info is supported, if the node sync from the first block, then the transacionINFOs of alll transactions can be queried.

**Comment by nateless** [comment]
@sunflower-sun yeah, my question was exactly from which block height transaction info is supported?

**Comment by panbhatt** [comment]
i think 2447335 onwards.

**Comment by nateless** [comment]
@ethan1844 @sunflower-sun `transactionInfo` does not provide sufficient information. For example how to get value of 32B tokens sent in this [trx](https://tronscan.org/#/transaction/5281eb99b3cab6b3cb4777e6b070df936f2cb068163fb8f3959a1cd88733a4c2)? How to get information whether this transaction is success or not, and if not hat is an error? How to get logs or internal transactions? They are also empty.
